google-cloud-firestore/pom.xml

```diff
@@ -131,8 +131,8 @@
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-all</artifactId>
-      <version>1.10.19</version>
+      <artifactId>mockito-core</artifactId>
+      <version>4.11.0</version>
```

Comment:
This is on par with java-storage's setup https://github.com/googleapis/java-storage/blob/6d4f851d96cac6c40f214e4d3de032e0500ba7a0/pom.xml#L108